menu.py

```diff
@@ -266,10 +266,15 @@ def show_menu():
           "11":{ "label": "Modify Num Epochs", "file": "custom-files/hyperparameters.json", "key": "num_epochs", "dtype" : "int"},
           "12":{ "label": "Modify Action Space", "file": "custom-files/model_metadata.json", "key": "action_space", "dtype" : "array"},
           "13":{ "label": "Modify Base Stack Name", "file": "custom-files/run.env", "key": "BASE_STACK_NAME", "dtype" : "string"},
-          "14":{ "label": "Set New Reward Function", "func": set_new_reward},
-          "15":{ "label": "Add IP Access", "func": add_ip},
-          "16":{ "label": "Run New Training", "func": run_training, "args": (False,)},
-          "17":{ "label": "Continue A Training", "func": run_training, "args": (True,)},
+          
+          "14":{ "label": "Modify Round Robin Advance Dist (Workers/Epsidoes-Between-Training)", "file": "custom-files/run.env", "key": "DR_TRAIN_ROUND_ROBIN_ADVANCE_DIST", "dtype" : "string"},
```

Comment:
please fix typo:
Epsidoes-Between-Training

also, check the typing on:
 DR_TRAIN_ROUND_ROBIN_ADVANCE_DIST type should be float?
 DR_TRAIN_MIN_EVAL_TRIALS type should be int?

Reply:
Took a second look and the type appears to no be used in case the file is .env - corrected anyway just in case I'm missing something else
Thank you!

Reply:
+1 to what @pablojr  said

Reply:
Should be okay now!